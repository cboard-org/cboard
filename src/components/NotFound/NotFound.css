.NotFound {
  height: 100%;
  padding: 1em;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: #eceff1;
  background: linear-gradient(to right, rgb(45, 22, 254), rgb(141, 92, 255));
}

.NotFound__codeError {
  font-size: 3rem;
  margin-bottom: 0.25rem;
}
