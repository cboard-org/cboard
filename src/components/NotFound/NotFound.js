import React from 'react';

import './NotFound.css';

const NotFound = () => (
  <div className="NotFound">
    <p className="NotFound__codeError">404</p>
    This page doesn't exist
  </div>
);

export default NotFound;
