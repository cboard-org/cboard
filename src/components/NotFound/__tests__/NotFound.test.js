import React from 'react';

import { shallowMatchSnapshot } from '../../../common/test_utils';
import NotFound from '../NotFound';

describe('NotFound tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<NotFound />);
  });
});
