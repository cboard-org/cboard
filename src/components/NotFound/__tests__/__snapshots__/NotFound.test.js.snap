// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound tests default renderer 1`] = `
<div
  className="NotFound"
>
  <p
    className="NotFound__codeError"
  >
    404
  </p>
  This page doesn't exist
</div>
`;
